applications/pelion_client/README.rst

```diff
@@ -0,0 +1,560 @@
+| Module                        | Related application events                                                                               | Comment                                                                                                                                |
++===============================+==========================================================================================================+========================================================================================================================================+
+| ``pelion``                    | - Sends ``pelion_state_event`` and ``pelion_create_objects_event``                                       | Brings up the Device Management Client object and interacts with it. Monitors the connection state to the Device Management Portal.    |
+|                               | - Reacts to ``net_state_event``                                                                          | Brings up the Device Management Client object and interacts with it. Monitors the connection state to the Device Management Portal.    |
++-------------------------------+----------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
+| ``pelion_fcc``                | Reacts to ``factory_reset_event``                                                                        | Initializes Pelion's Factory Configurator Client (FCC) and initiates the developer flow mode.                                          |
++-------------------------------+----------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
+| ``pelion_trace``              | -                                                                                                        | If enabled, it initializes debug traces in the Pelion DM library.                                                                      |
++-------------------------------+----------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
+| ``net_ot``                    | - Send ``net_state_event``                                                                               | Initialize the network back end and tracks the network connection state.                                                               |
```

Comment:
```suggestion
| ``net_ot``                    |  Send ``net_state_event``                                                                               | Initializes the network back end and tracks the network connection state.                                                               |
```

Reply:
This refers to two modules at the same time. I'll keep the plural.